modules/standard/Sys.chpl

```diff
@@ -246,6 +246,16 @@ module Sys {
   extern proc sys_strerror(error:err_t, ref string_out:c_string):err_t;
 
   extern proc sys_readlink(path:c_string, ref string_out:c_string):err_t;
+
+  /*Check if the environment variable `name` is defined or not.
+    If `name` is defined then return 1 and update `string_out` 
+    as the value of environment variable
+    else the function returns 0.
+
+    :arg name: name of environment variable
+    :arg c_string: store the value of `name` environment variable if defined
+    :returns: `int` 1 if `name` is defined and 0 if not
```

Comment:
Technically, it returns a `c_int` rather than an `int`.  I might just drop the mention of the type here for simplicity, or else use the `:rtype:` tag to indicate that it's `c_int`.